hoover/search/views.py

```diff
@@ -119,7 +120,14 @@ def search(request):
         })
 
 
-@limit_user
+def thumbnail_rate(group, request):
+    has_thumbnail = re.search(r'/thumbnail/\d{3}/.jpg$', request.path)
```

Comment:
Why the `{3}` here?

Also, let's pin the start of the URLs too into the regex. not just the end.